.gitignore

```diff
@@ -1,5 +1,4 @@
 .vscode
-build
```

Comment:
Any reason to include the build directory?

Reply:
Only needed it when using my version in npm